### ISSUE TITLE
Change default units

### ISSUE THREAD

**Issue by tylerflex** [flexcompute/tidy3d#175]
THz
picoseconds

Might be something for next release only, but marking it.

**Comment by momchil-flex** [comment]
I guess better do it earlier rather than later. Come to think of it there shouldn't be much that needs to change on the backend either once the constants are redefined... I'll give it a try.

**Comment by tylerflex** [comment]
- [ ] Change units in the `Field()` automatically based on the default unit values.
- [ ] Change all notebooks, docstrings, examples.
- [ ] Redefine constants in `tidy3d.constants`
- [ ] Change any conversion that happens, say, in the fitter (Hz -> eV)

**Comment by tylerflex** [comment]
@momchil-flex are we considering this still? it would be quite a big change but if we're going to do it, should be before 2.0. I'm leaning towards just living with our current units. Thoughts? Close if you agree

**Comment by momchil-flex** [comment]
I agree, this should have been done a long time ago if ever, too many other things now.
